dbt/adapters/athena/impl.py

```diff
@@ -397,9 +399,22 @@ def clean_up_table(self, relation: AthenaRelation) -> None:
         if table_location := self.get_glue_table_location(relation):
             self.delete_from_s3(table_location)
 
+    def quote(self, identifier: str) -> str:
+        return f"{self.quote_character}{identifier}{self.quote_character}"
+
     @available
-    def quote_seed_column(self, column: str, quote_config: Optional[bool]) -> str:
-        return str(super().quote_seed_column(column, False))
+    def quote_seed_column(
+        self, column: str, quote_config: Optional[bool], quote_character: Optional[str] = None
+    ) -> str:
+        if quote_character:
+            old_value = self.quote_character
```

Comment:
Could it be shared using a MixinClass with what is done in the `AthenaRelation` class ?

Reply:
@davidbridgwood did you see this comment? 

@Jrmyy using MixinClass could work, but I would say we can crate a new class, and reuse the class functionally both here and in AthenaRelation, as implementation is really similar - but if you have a more concrete suggestion maybe you can provide an example?